resharper/src/resharper-unity/CSharp/Daemon/Stages/Analysis/DrawGizmoAttributeProblemAnalyzer.cs

```diff
@@ -47,49 +47,43 @@ private void CheckMethodDeclaration(IMethodDeclaration methodDeclaration, IAttri
         {
             if (methodDeclaration == null)
                 return;
-
+            
             var predefinedType = myPredefinedTypeCache.GetOrCreatePredefinedType(element.GetPsiModule());
-            var gizmoTypeName = TypeFactory.CreateTypeByCLRName("UnityEditor.GizmoType", predefinedType.Module);
-            var componentName = TypeFactory.CreateTypeByCLRName("UnityEngine.Component", predefinedType.Module);
+            var gizmoType = TypeFactory.CreateTypeByCLRName("UnityEditor.GizmoType", predefinedType.Module);
+            var componentType = TypeFactory.CreateTypeByCLRName("UnityEngine.Component", predefinedType.Module);
+          
```

Comment:
The last comment is outdated, but what I meant was:

```
TypeFactory.CreateTypeByCLRName(KnownTypes.Component, ...)
```

You'll have to add `KnownTypes.GizmoType`. Using this string overload will create an instance of `ClrTypeName` with what you pass in. Using `KnownTypes` just shares those instances.